.github/workflows/libfuzzer.yaml

```diff
@@ -0,0 +1,170 @@
+name: Libfuzzer
+"on":
+  push:
+    branches:
+      - main
+      - prerelease_test
+  pull_request:
+    paths: .github/workflows/libfuzzer.yaml
+
+jobs:
+  fuzz:
+    name: Compression
+    runs-on: ubuntu-22.04
+    env:
+      PG_VERSION: 15.2
```

Comment:
Should this be strict to a single pg version? What about use the gh_matrix_builder.py to this?

Reply:
Since we are mostly testing a very narrow subset of our own code here, the Postgres version doesn't change much, so I decided to keep it simple for now.